clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -317,9 +317,9 @@ func (w *waitForControlledPodsRunningMeasurement) handleObjectLocked(oldObj, new
 		operationTimeout *= 2
 	}
 	time.AfterFunc(operationTimeout, func() {
-		checker.terminate(true)
+		c.terminate(true)
 	})
-	w.checkerMap[key] = checker
+	((map[string]checker.Checker)(w.checkerMap))[key] = c
```

Comment:
We don't want to do this.
If this case is really needed, you need to add a method on Checker map like SetChecker(key, checker)

Reply:
Done.